indico/modules/users/templates/personal_data.html

```diff
@@ -15,11 +15,63 @@
+                                               record-name="{{ user.first_name }} {{ user.last_name }}"
+                                               delete-url="{{ url_for('users.user_delete', user) }}"
+                                               redirect-url="{{ url_for('users.users_admin') }}">
+                                {% trans %}Once deleted, the following will happen:{% endtrans %}
+                                <div style="padding-top: 15px; padding-bottom: 15px">
+                                    <ul class="ui list" style="list-style: none; padding-left: 0; margin-top: 0">
+                                        <li class="item">
+                                            <i class="minus circle icon"></i>
+                                            <div class="content">
+                                                <span>
+                                                    {% trans first_name=user.first_name %}
+                                                        <strong>{{ first_name }}</strong> will no longer be able to access Indico.
+                                                    {% endtrans %}
+                                                </span>
+                                            </div>
+                                        </li>
+                                        <li class="item">
+                                            <i class="times circle outline icon"></i>
+                                            <div class="content">
+                                                <span>
+                                                    {% trans first_name=user.first_name %}
+                                                        <strong>{{ first_name }}</strong> will be removed from all areas of Indico - this does not affect
+                                                        their presence in events as a speaker or other role; they will still be listed in such
+                                                        capacities where applicable.
+                                                    {% endtrans %}
+                                                </span>
+                                            </div>
+                                        </li>
+                                        <li class="item">
+                                            <i class="trash alternate outline icon"></i>
+                                            <div class="content">
+                                                <span>
+                                                    {% trans first_name=user.first_name %}
+                                                        Where it is not possible to delete <strong>{{ first_name }}</strong>, they will be anonymized and
+                                                        all personal data associated with the user will be removed from Indico.
```

Comment:
This is also misleading and contradicting with the previous point. For instance, Registration data will be unlinked from the user account but it will NOT be anonymized and for good reasons. As far as I understand, the registration data is required for the event manager for organizational reasons and it's not up to the registrant to have it deleted.